src/main/java/hudson/plugins/git/GitPublisher.java

```diff
@@ -579,15 +579,15 @@ public static final class NoteToPush extends PushConfig {
         private String noteNamespace;
         private boolean noteReplace;
 
-        public String getnoteMsg() {
+        public String getNoteMsg() {
```

Comment:
Unfortunately for binary backwards compatibility the old method signatures needs to be kept and `@Deprecated`.

Also shouldn't there be needed changes in the config.jelly for the signature change?

Reply:
Thanks @rsandell, will add the old method signatures back. And good catch with the jelly sections.

Reply:
Reinstated the old methods.

I checked the `config.jelly` and the fields are referenced so it seems jelly does not need the official getters, just the constructor: https://github.com/jenkinsci/git-plugin/blob/9f17f9ab5d9ee2479f9fa71dc2cc3b95b7e2ee40/src/main/resources/hudson/plugins/git/GitPublisher/config.jelly#L75-L89